### ISSUE TITLE
See Also Relationships across collections

### ISSUE THREAD

**Issue by flapka** [ycba-cia/blacklight-collections2#276]
This issue also follow an email exchange between @edgartdata, @BaylaArietta, and @flapka. Emmanuelle said: 

> With the goal to provide more contextual information for our objects online, we also talked about defining how our objects relate to other objects in our collections and in other collections, such the Sowerby drawings to the Peabody collection. If you could start to define these relationships (this might be a brief list), and what would be needed to publish them online to our BlackLight that would be great.

**Comment by flapka** [comment]
Possible relationships for RB (pointing internally or externally) include things like: based on, adaptation of, inspired by, contained in, illustrations for, reproduction of, reprint of, accompanied by, bound with. 

I'm sure there are other good ones that I'm forgetting

**Comment by edgartdata** [comment]
Agenda item for 7/30 meeting.

**Comment by edgartdata** [comment]
See also relationships scoped by Reference Library (see examples in the attached below in Word doc):

- Preparatory drawing for a painting
- Finished painting that has preparatory drawings in and outside the YCBA collection 
- Painting in the YCBA collection that is very similar to another painting in YCBA Collection 
- Painting outside the YCBA collection that is similar to a YCBA Painting (Allegory of the Tudor Succession)
- A citation that includes discussion that “might” be about a YCBA painting (unclear from the entry if it is the YCBA painting) (Remaining Pictures and unfinished sketches of Sir Thomas Lawrence – June 18, 1831 Christie’s auction) 
- A print after a painting 

Examples: [See Also examples.docx](https://github.com/ycba-cia/blacklight-collections2/files/6895948/See.Also.examples.docx)

From Kraig: In the cases where I refer to an object outside the YCBA I envision having a see-also bib reference that links to the online object record at another institution, as well as book/article citations. I see these see-also references as being able to link scholarship about very similar objects as well as add bib information to YCBA objects that have very little scholarship devoted to them.  
I think ideally, in the online bibliography, these citations would be separate from the main citations and be displayed afterward with a new heading. I feel it may be too complicated to intermingle them with the others, and by having them fall beyond the main citations, it connotates that these are really different in content.

**Comment by edgartdata** [comment]
The next steps I see for this to go forward (all, please record your questions/comments/suggestions/concerns below):

- Once we identify the 2 or 3 see also relationships that we want to document systematically, we can put our recommendations to Martina (she might narrow this down further)
- We need to prioritize which works of art these 2 or 3 see also relationships apply to (probably the list of 50 paintings or so that Ed is working on so that this project leverages research from curatorial as well?) (@flapka Will there be priority items in RBMS collections for this project?)
- Structure the data entry in TMS with (and coordinate with RBMS/Voyager):

1. We will need to have a field in the TMS Bib Module records that identify which kind of See Also relationship is used between objects every time. I am thinking the Attributes field would be perfect for that but I would like to test it first.
2. For the bibliography illustrating or discussing how 2 objects are connected by a specific See Also relationship, we will want to flag that the citation is for that scenario, and not for a classic citation as your team has been documenting them in the past. 

So in short, we need 2 new TMS fields with pick lists for :
-  the different types of See Also relationship
- to flag citations that are for non YCBA objects

**Comment by flapka** [comment]
> (@flapka Will there be priority items in RBMS collections for this project?)
Nothing high priority, that I'm aware of. RB has a dozen or two records that reference objects in P&D, usually where original drawings (in P&D) are the sources used for published illustrations for a book in RB. Here's an example of that:  https://collections.britishart.yale.edu/catalog/orbis:6098270

It'd be nice to come up with formal (and consistent) treatment of such relationships, especially if it leads to tighter connections in Blacklight.

**Comment by edgartdata** [comment]
7/30/21 meeting notes:

For now we will focus on these 3 See Also relationships:

**- Paintings in YCBA collection that have preparatory drawings in the YCBA collection and in other collections:** 
Example: 
![YCBA Girl Holding a Cat B1993 30 12](https://user-images.githubusercontent.com/4093644/127672878-dc6f6426-d4bd-409e-9a93-65964271ed8b.png)
Young Woman Holding a Cat (oil on canvas; B1993.30.12; https://collections.britishart.yale.edu/catalog/tms:1400) has 2 preparatory drawings: 

   1. one in the YCBA collection: Girl in an Apron with a Cat on her Lap (drawing; B1993.30.98; https://collections.britishart.yale.edu/catalog/tms:4188)
![YCBA Girl in an Apron with a Cat on her Lap B1993 30 98](https://user-images.githubusercontent.com/4093644/127672429-7df3a316-6173-4ea3-9b2f-726fe0f82fdd.png)

   2. another one in the Metropolitan Museum collection: Girl with a Cat (drawing; 1991.158; https://www.metmuseum.org/art/collection/search/485715)
![Met Museum Girl with a Cat 1991 158](https://user-images.githubusercontent.com/4093644/127672567-9e0dff0a-bc24-4761-8d47-b77e3afb5613.png)

**YCBA painting is very similar to another painting in YBCA collection or external collection:** 
![YCBA Tudor Allegory B1974 3 7](https://user-images.githubusercontent.com/4093644/127674493-443351d7-c3b4-4f05-babe-62fe7d3f96a3.png)
An Allegory of the Tudor Succession: The Family of Henry VIII (Oil on panel; B1974.3.7; https://collections.britishart.yale.edu/catalog/tms:168)
has 2 other versions that are similar in composition and significant enough for mentioning here (in other words, we would not want to attempt to capture ALL versions/iterations of an image but only the significant ones):

   1. The family of Henry VIII: an allegory of the Tudor succession (National Museum Wales; Item Number NMW A 564; https://museum.wales/collections/online/object/41bb6c94-b7f3-3caf-ad10-d63cb2935df6/The-family-of-Henry-VIII-an-allegory-of-the-Tudor-succession/?field0=string&value0=The%20Family%20of%20Henry%20VIII%20An%20Allegory%20of%20the%20Tudor%20Succession%20-%20HEERE%2C%20Lucas%20De.&field1=with_images&value1=on&index=0)
![National Museum Wales Tudor Allegory NMW A564](https://user-images.githubusercontent.com/4093644/127674777-4d2ada7c-bd42-4449-9b3d-051486ecd44d.png)

   2. Tudor Allegory print (British Museum; Museum Number 1842,0806.373; https://www.britishmuseum.org/collection/object/P_1842-0806-373)
   
![British Museum Tudor Allegory print 1842,0806 373](https://user-images.githubusercontent.com/4093644/127675889-0a81f4de-d810-4ae2-a504-14bc9520a8ee.png)

**- A YCBA print after a painting in YCBA collection or external collection:**
Joseph Wright of Derby, The Blacksmith's Shop (oil on canvas; Accession Number: B1981.25.712; https://ycba-collections-dev.herokuapp.com/catalog/tms:5042)
![Wright of Derby Blacksmith's Shop B1981 25 712](https://user-images.githubusercontent.com/4093644/127676775-cf5b387f-f407-4e5f-b09d-dfa148874009.png)

Print made by Richard Earlom after Joseph Wright of Derby, A Blacksmith's Shop (Mezzotint; B1977.14.14589; https://ycba-collections-dev.herokuapp.com/catalog/tms:40583)
![Richard Earlom Blacksmith's Shop print B1977 14 14589](https://user-images.githubusercontent.com/4093644/127677120-1a0aa473-b0f6-4886-b6fb-0c13587fccee.png)

**Comment by edgartdata** [comment]
Several next steps:
- work out the TMS fields for the necessary metadata
- work out the LIDO XML elements to vend the above metadata flags to BlackLight
- work out more examples for each of these 3 relationships
- recommend process for curatorial approval of suggested relationships (monthly list submitted to curatorial?)
- think about potential design for this additional content in the BlackLight records (See Also tab with a section for each of these relationships? with appropriate citations in each of these sections?)

**Comment by edgartdata** [comment]
Question about the relationship 'A YCBA print after a painting in YCBA collection or external collection': the print and the painting could be by the same artist or different artist, correct?

**Comment by edgartdata** [comment]
Several additional P&D examples provided by Amelia Giordano: 
[Connections between YCBA and external works_Amelia Giordano_August 2021.docx](https://github.com/ycba-cia/blacklight-collections2/files/7092256/Connections.between.YCBA.and.external.works_Amelia.Giordano_August.2021.docx)

**Comment by flapka** [comment]
Posting here to:
1. ask if we want more discussion on the next steps outlined above
2. add an example, provided by Charlotte Lefland. A researcher pointed out to us this week that our print [B1977.14.14924](https://collections.britishart.yale.edu/catalog/tms:43746) also appears in a book in RBM ([BX5194 W54 1727](https://collections.britishart.yale.edu/catalog/orbis:764463)). My hunch is that our collections include hundreds (or thousands?) of examples of this category of relationship.

**Comment by edgartdata** [comment]
Another example: the 2 Bardwell paintings https://collections.britishart.yale.edu/catalog/tms:83373 and https://collections.britishart.yale.edu/catalog/tms:83373 are related to the sketchbook by Bardwell https://collections.britishart.yale.edu/catalog/tms:58388

Right now the paintings are linked to the sketchbook via the bibliography. I am not sure that’s the best way to do that (also the sketchbook does not currently link to the paintings).

**Comment by KraigBinkowski** [comment]
In the Bardwell case - this is a kind of scrap book that puts together print reproductions of the work of art the Center owns. So a little bit of a different and unique situation. Ed specifically asked that these reproductions in the scrapbook be linked to the work of art via the bibliography. I don't anticipate linking works of art to other sketchbooks at this time unless we can expand the categories of the bibliography to include "see also" or "related resources".
